Cargo.toml

```diff
@@ -1,3 +1,15 @@
+[package]
+name = "parsec"
+version = "0.1.0"
```

Comment:
Are we obliged to provide a version field here ?

I would be simpler not to (given the version is already handling in `parsec/_version.py`)

Alternatively we could just set `version = "0.0.0"` with a comment explaining this is a dummy version because it is mandatory

Last possibility is to modify `releaser.py` to update this, but it's much more complicated for no real benefit so we should avoid this

Reply:
done, rust require the field `version`

So the field is set to `0.0.0` with a comment